framework/assets/yii.js

```diff
@@ -46,17 +46,18 @@ window.yii = (function ($) {
         /**
          * List of JS or CSS URLs that can be loaded multiple times via AJAX requests.
          * Each item may be represented as either an absolute URL or a relative one.
-         * Each item may contain a wildcart matching character `*`, that means one or more
+         * Each item may contain a wildcard matching character `*`, that means one or more
          * any characters on the position. For example:
-         *  - `/css/*.js` will match any file ending with `.js` in the `css` directory of the current web site
+         *  - `/css/*.css` will match any file ending with `.css` in the `css` directory of the current web site
```

Comment:
Having `js` files in `css` folder is kind of weird. If it was written consciously - it's not a good example, otherwise - it's just a typo.